data/presets/shop/gift.json

```diff
@@ -5,10 +5,13 @@
         "area"
     ],
     "terms": [
-        "souvenir"
+        "tourist trap"
```

Comment:
Hah, that might as well be a term for most `tourism=attraction`s while we’re at it.

Reply:
Yeah but the scope of this PR is just shops. It's big enough already IMO. I included this as a term so that one can start typing "tourist" and also find this preset.